src/lib/DownloadReachability.js

```diff
@@ -55,7 +55,7 @@ class DownloadReachability {
         // 2. or starts with a custom prefix and ends with boxcloud.com
         return (
             !downloadUrl.startsWith(DEFAULT_DOWNLOAD_HOST_PREFIX) &&
-            (!!downloadUrl.match(NUMBERED_HOST_PREFIX_REGEX) || downloadUrl.indexOf(PROD_CUSTOM_HOST_SUFFIX) !== -1)
+            (!!downloadUrl.match(VALID_CUSTOM_HOST_PREFIX_REGEX) || downloadUrl.indexOf(PROD_CUSTOM_HOST_SUFFIX) !== -1)
```

Comment:
Looking at this again, maybe we could simplify to say that it just shouldn't start with the default host prefix and ends with our boxcloud suffix? This will guarantee it comes from box and still validates that it's custom.

Reply:
Sounds good let me change that